upgrade/scripts/common/ncn-rebuild-common.sh

```diff
@@ -80,120 +80,133 @@ else
 fi
 
 
-state_name="SHUTDOWN_SERVICES"
+state_name="SET rd.live.dir AND rd.live.overlay.reset"
 state_recorded=$(is_state_recorded "${state_name}" ${target_ncn})
 if [[ $state_recorded == "0" ]]; then
+    rm -f ${basedir}/standdown.sh
     echo "====> ${state_name} ..."
     {
-
-    if [[ $target_ncn == ncn-s* ]]; then
-        # Nothing to do.
-        :
-    elif [[ $target_ncn == ncn-m* ]]; then
-    cat <<'EOF' > standdown.sh
-    echo 'unmounting USB(s) ... '
-    usb_device_path=$(lsblk -b -l -o TRAN,PATH | awk /usb/'{print $2}')
-    usb_rc=$?
-    set -e
-    if [[ "$usb_rc" -eq 0 ]]; then
-      if blkid -p $usb_device_path; then
-        have_mnt=0
-        echo 'unmounting discovered USB mountpoints ... '
-        for mnt_point in /mnt/rootfs /mnt/sqfs /mnt/livecd /mnt/pitdata; do
-          if mountpoint $mnt_point; then
-            have_mnt=1
-            umount -v $mnt_point
-          fi
-        done
-        if [ "$have_mnt" -eq 1 ]; then
-          echo 'ejecting discovered USB: [$usb_device_path]
-          eject $usb_device_path
-        fi
-      fi
-    fi
-    umount -v /var/lib/etcd /var/lib/sdu || true
-    
-    echo 'Deactivating disk boot entries to force netbooting for rebuilding ... '
-    efibootmgr # print before
-    efibootmgr | grep '(UEFI OS|cray)' | awk -F'[^0-9]*' '{print $0}' | sed 's/^Boot//g' | awk '{print $1}' | tr -d '*' | xargs -r -i efibootmgr -b {} -B
-    efibootmgr # print after
-    echo 'Setting next boot to PXE ... '
-    ipmitool chassis bootdev pxe options=efiboot
-EOF
-    else
-    cat <<'EOF' > standdown.sh
-    lsblk | grep -q /var/lib/sdu
-    sdu_rc=$?
-    vgs | grep -q metal
-    vgs_rc=$?
-    set -e
-    echo 'Disabling and stopping kubernetes and containerd daemons ... '
-    systemctl disable kubelet.service || true
-    systemctl stop kubelet.service || true
-    systemctl disable containerd.service || true
-    systemctl stop containerd.service || true
-    umount -v /var/lib/containerd /var/lib/kubelet || true
-    if [[ "$sdu_rc" -eq 0 ]]; then
-      umount -v /var/lib/sdu || true
-    fi
-        
-    echo 'Deactivating disk boot entries to force netbooting for rebuilding ... '
-    efibootmgr # print before
-    efibootmgr | grep '(UEFI OS|cray)' | awk -F'[^0-9]*' '{print $0}' | sed 's/^Boot//g' | awk '{print $1}' | tr -d '*' | xargs -r -i efibootmgr -b {} -B
-    efibootmgr # print after
-    echo 'Setting next boot to PXE ... '
-    ipmitool chassis bootdev pxe options=efiboot
-EOF
+    if [ -z "${CSM_RELEASE}" ]; then
+        echo 2>& "CSM_RELEASE is not set!"
+        exit 1
     fi
-    chmod +x standdown.sh
-    scp standdown.sh $target_ncn:/tmp/standdown.sh
-    ssh $target_ncn '/tmp/standdown.sh'
-    } >> ${LOG_FILE} 2>&1
+    csi handoff bss-update-param --delete rd.live.dir --limit $TARGET_XNAME
```

Comment:
For consistency, since I think we generally use curly braces, even when not necessary.

```suggestion
    csi handoff bss-update-param --delete rd.live.dir --limit ${TARGET_XNAME}
```